### PR TITLE
Issue 280: sustainability child pages - Biodiversity

### LINKED ISSUE #280: Page: sustainability child pages
Migrating: https://www.sunstar.com/sustainability/environment/biodiversity/
https://www.sunstar.com/sustainability/environment/products/

### PR DESCRIPTION
**Pull request by shrotia**
1. Introduced a new column variant for Row Cards, allowing the display of long cards with Image & Text side by side.
2. Added variant for Heading Divider within columns.

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #280 

Test URLs:
- Original: https://www.sunstar.com/sustainability/environment/biodiversity/
- Before: https://main--sunstar--hlxsites.hlx.live/sustainability/environment/biodiversity
- After: https://issue-280-v2--sunstar--hlxsites.hlx.live/sustainability/environment/biodiversity

- [x] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------

-  Introduced a new column variant for Row Cards, allowing the display of long cards with Image & Text side by side. 
- Added variant for Heading Divider within columns.

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -161,6 +161,13 @@
   max-width: 1200px;
 }
 
+.columns.heading-divider h2,
+.columns.heading-divider h3,
+.columns.heading-divider h4 {
+  border-bottom: 1px solid #3c4650;
```
Comment (shrotia) [inline blocks/columns/columns.css:167]: @bosschaert commented: The color is available as a variable here: https://github.com/hlxsites/sunstar/blob/main/styles/styles.css#L59

Comment (bosschaert) [reply blocks/columns/columns.css:167]: Correct, so you reference it like this: `var(--light-black)`

**Review by sdmcraft — APPROVED** [review]
LGTM. Please add these to the block library for reference.

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -161,6 +161,13 @@
   max-width: 1200px;
 }
 
+.columns.heading-divider h2,
```
Comment (mkbansal1) [inline blocks/columns/columns.css:164]: @shrotia These changes are taken care as part of PR 
https://github.com/hlxsites/sunstar/pull/351

Please see this page for authoring
https://345-column-block-enhancement--sunstar--hlxsites.hlx.page/sustainability/environment/climate-resource-waste

**Comment by shrotia** [comment]
For the PSI, created an issue: https://github.com/hlxsites/sunstar/issues/355

File: blocks/columns/columns.css (hunk 2 above)
Comment (shrotia) [reply blocks/columns/columns.css:164]: oops, I missed this comment before merging. Let me reconcile

**Pull request merged**